src/saving/SaveList.tscn

```diff
@@ -128,6 +129,11 @@ WindowTitle = "SELECTED_SAVE_IS_INCOMPATIBLE_PROTOTYPE_PROMPT"
 HideCancelButton = true
 DialogText = "LOAD_INCOMPATIBLE_PROTOTYPE_WARNING"
 
+[node name="ErrorDialog" parent="MarginContainer/VBoxContainer" instance=ExtResource( 4 )]
+margin_top = 4.0
+margin_right = 1270.0
+margin_bottom = 4.0
```

Comment:
The consequence of doing all of the things above is that we have a static error text, hence it's more appropriate to set the error dialog's title and message from the inspector instead of setting it programmatically. With one catch: we fill out a translation key instead of plain English text, for more info see here: [Working With Translations](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/working_with_translations.md).

Reply:
I changed the ErrorMessages to CustomConfirmationDialog, and added the title and description keys in the en.po file, though I'm a bit worried I broke something when I added them there

Reply:
It looks fine from a cursory look and looks okay now overall. I'll check back tomorrow (within my timezone).